src/back/spirv/parser.rs

```diff
@@ -0,0 +1,1545 @@
+/*! Standard Portable Intermediate Representation (SPIR-V) backend !*/
+use super::layout::{Instruction, Module};
+use crate::FastHashMap;
+use spirv::*;
+
+trait LookupHelper<T> {
+    type Target;
+    fn lookup_id(&self, handle: &crate::Handle<T>) -> Option<Word>;
+    fn lookup_handle(&self, word: &Word) -> Option<crate::Handle<T>>;
```

Comment:
any reason the `&Word` here is by reference? and same for `&crate::Handle`

Reply:
Code is removed